test/node/module/authenticate/setClientCert.test.js

```diff
@@ -182,7 +182,7 @@ describe('Connection module', () => {
       const auth = new Auth()
         .setPasswordAuth(paramPasswordAuth)
         .setClientCert({cert: undefined, password: certPass});
-      const conn = new Connection(common.DOMAIN, auth);
+      const conn = new Connection({domain: common.DOMAIN, auth: auth});
       const appModule = new App(conn);
```

Comment:
App module will be changed to single object params. Update App module first, then update this PR